mesheryctl/internal/cli/root/system/update.go

```diff
@@ -29,8 +29,8 @@ import (
 // updateCmd represents the update command
 var updateCmd = &cobra.Command{
 	Use:   "update",
-	Short: "Pull new Meshery images from Docker Hub.",
-	Long:  `Pull Docker Hub for new Meshery container images and pulls if new image version(s) are available.`,
+	Short: "Pull new Meshery images/manifest files and operator manifests.",
+	Long:  `Pull new Meshery container images/manifests and operator manifests from Github/Docker Hub and pull if new version(s) are available.`,
```

Comment:
Should we need to specify about operator manifests? I think just specifying images and manifests should be enough. Keep the user above a level of abstraction.

Reply:
agreed @navendu-pottekkat 
Done that in the latest commit.